src/Sarif.FunctionalTests/SarifConverterTestData/AndroidStudio/AndroidStudio_issueLog1_raw.xml.sarif

```diff
@@ -30,21 +60,7 @@
                   "startLine": 10
                 }
               },
-              "logicalLocation": [
-                {
-                  "name": "app",
-                  "kind": "android-module"
-                },
-                {
-                  "name": "microsoft.androidstudioformatexample",
-                  "kind": "package"
-                },
-                {
-                  "name": "microsoft.androidstudioformatexample.ApplicationTest ApplicationTest()",
-                  "kind": "method"
-                }
-              ],
-              "fullyQualifiedLogicalName": "app\\microsoft.androidstudioformatexample\\microsoft.androidstudioformatexample.ApplicationTest ApplicationTest()"
+              "logicalLocation": "app\\microsoft.androidstudioformatexample\\microsoft.androidstudioformatexample.ApplicationTest ApplicationTest()"
```

Comment:
I think two things are going on:
1. The "thing we were going to use where possible" is indeed the thing we were calling `fullyQualifiedLogicalName`, but I thought we agreed to change its name to `logicalLocation`.
2. The "thing we were only going to use when necessary" is the new thing I added, called `logicalLocationKey`.

Do you object to changing the name from `fullyQualifiedLogicalName` to to `logicalLocation`? I like the latter better.